inclusive-design/idrc
# GH-459: Improve the authoring experience (resolves #451, #459)

## Description

Improve the authoring experience for adding/updating contents. Now the content in the preview panel of Netlify CMS should match the corresponding live content.

## Steps to test

1. Login and go to /admin/ page;
2. Open any content or create a new content;
3. The content in the preview pane should match the live content.
